client/src/utils/CSSHelp.test.ts

```diff
@@ -0,0 +1,81 @@
+/* global describe it expect */
+import { cssString } from './__fixtures/curriculum-helpers-css';
+import CSSHelp from './CSSHelp';
+
+describe('CSSHelp should pass all tests..', () => {
```

Comment:
```suggestion
describe('css-help', () => {
```
Could you also lower case the filenames?  We're not very good about this, but we should try to always use lower case.